### DISCUSSION TITLE
Is there a particular reason why physical iOS devices are not supported?

### DISCUSSION THREAD

**Discussion by dudek-j** [mobile-dev-inc/Maestro#240] [Q&A] [answered]
Glossing over it [idb](https://github.com/facebook/idb ) supports physical devices. 
Is there any concrete reason why this doesn't carry over into maestro?

**Comment by dudek-j** [comment, marked as answer]
Upon further investigation it seems like idb doesn't support XCUITest based tests on devices which is probably the reason here.

**Reply by dmitry-zaitsev** [reply]
@dudek-j that is correct, that said we are exploring options for making that possible

**Reply by papaschmidt** [reply]
@dmitry-zaitsev -  Has there been any updates on this since October?  My company is exploring using Maestro at the moment and currently the lack of real-device support for iOS is a major negative for us for what otherwise looks like has everything we want.

**Reply by boomer30** [reply]
@dmitry-zaitsev we would be very curious to hear any information regarding support for iOS physical devices as well. We are currently evaluating Maestro and very much like what we see so far. The one kicker is not having support for iOS devices

**Reply by dudek-j** [reply]
During my research it seemed like Appium is still the way to go. The only cross platform testing framework with on device support I was able to find. Hopefully Maestro can add iOS physical device at some point in the future.

**Reply by boomer30** [reply]
@dudek-j thanks for the feedback. Our apps are Flutter based. Please correct me if I am wrong, but I believe Appium suffers from the same challenges as Firebase Test Labs in that you cannot test any native plugins w/in your Flutter app. That is where the appear of Maestro comes in for us.

**Reply by dudek-j** [reply]
I have only had the chance to try Appium with native iOS and Android apps. The framework uses accessibility tags to make its way around and interact with the application, which if I am not mistaken, is the way Maestro operates. Unfortunately my Flutter knowledge is very limited so I don't have any useful input here.

**Reply by NilPascual** [reply]
Any news on the topic, also interested in using the tool but no IOS physical device support its a blocker for us.
